packages/moleculer-db-adapter-prisma/schema.prisma

```diff
@@ -0,0 +1,17 @@
+generator client {
```

Comment:
Does this file exist only for example? Can we move it into the examples folder?

Reply:
yes, I think so

Reply:
After the moving, the tests failed: https://github.com/moleculerjs/moleculer-db/runs/7949047525?check_suite_focus=true

Reply:
This happened due to the fact that when using a custom schema file location, it is necessary to start the generation with an additional argument. The test is launched at a higher level and prehooks from packages are not processed. I propose to return the schema to the root and then the post-install hook will perform the generation.

Reply:
@icebob